.drone.yml

```
@@ -1,120 +1,149 @@
 ---
 kind: pipeline
 type: kubernetes
-name: deploy
+name: deploy tag
+trigger:
+  event:
+    - tag
+  branch:
+    - main
+
+steps:
+  - name: checkout
+    image: quay.io/ukhomeofficedigital/hocs-helm-kubectl:1.0.0
+    commands:
+      - git fetch --tags
+      - git checkout ${DRONE_TAG}
+      - git submodule update --init --recursive
+
+  - &deploy
+    name: cs-dev
+    image: quay.io/ukhomeofficedigital/hocs-helm-kubectl:1.0.0
+    environment:
+      CHART_NAME: hocs-notify
+      KUBE_CLUSTER: acp-notprod
+      KUBE_NAMESPACE: cs-dev
+      KUBE_SERVER: https://kube-api-notprod.notprod.acp.homeoffice.gov.uk
+      KUBE_TOKEN:
+        from_secret: cs_notprod
+      VERSION: ${DRONE_TAG}
+    commands:
+      - ./ci/helm/helm-deploy.sh --values ./helm/values-notprod.yaml --values ./helm/values-dev.yaml
+    depends_on:
+      - checkout
+
+  - <<: *deploy
+    name: wcs-dev
+    environment:
+      KUBE_NAMESPACE: wcs-dev
+    depends_on:
+      - cs-dev
```

Are you sure you want these to deploy sequentially instead of in parallel?
```suggestion
    depends_on:
      - checkout
```